src_c/music.c

```diff
@@ -496,6 +496,66 @@ music_queue(PyObject *self, PyObject *args, PyObject *keywds)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+music_get_metadata(PyObject *self, PyObject *args, PyObject *keywds)
+{
+    PyObject *meta_dict;
+    Mix_Music *music = current_music;
+
+    PyObject *obj = NULL;
+    char *namehint = NULL;
+    static char *kwids[] = {"filename", "namehint", NULL};
+
+    if (!PyArg_ParseTupleAndKeywords(args, keywds, "|Os", kwids, &obj,
+                                     &namehint))
+        return NULL;
+
+    MIXER_INIT_CHECK();
+
+    if (obj) {
+        music = _load_music(obj, namehint);
+        if (!music) {
+            return NULL;
+        }
+    }
+    else if (namehint) {
+        PyErr_SetString(
+            PyExc_RuntimeError,
```

Comment:
Runtime error is strange here. Maybe a pygame.error or a ValueError?

Reply:
I wasn't sure what type to use since the issue is that one of the optional arguments is required to use the other one, however, it's possible to pass only the other one since they're both optional. A `pygame.error` could cover this case I guess.